packages/microsoft_sqlserver/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,65 @@
+---
+description: Pipeline to parse Microsoft SQL Server logs
+processors:
+- set:
+    field: ecs.version
+    value: 8.3.0
+- set:
+    field: event.ingested
+    value: '{{_ingest.timestamp}}'
+- grok:
+    field: message
+    patterns:
+    - '%{MSSQL_DATE:date} %{DATA:mssql.log.origin} [ ]*%{GREEDYDATA:msg_temp}'
+    - '%{GREEDYDATA:msg_temp}'
+    pattern_definitions:
+      MSSQL_DATE: '%{DATA} %{DATA}'
+      GREEDYDATA: |-
+          (.|
+          |	)*
```

This looks exceedingly brittle to accidental editor damage to me. Can you use escape sequences, please.
```suggestion
      GREEDYDATA: '(.|\n)*'
```
Note that the `\t` is already matched by the `.`, only the newline is special (this change is confirmed locally).